runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java

```diff
@@ -304,4 +317,57 @@ private void removeOverheadsIndicators()
 		infoBoxManager.removeIf(entry -> entry instanceof PrayerCounter
 			&& ((PrayerCounter) entry).getPrayerType().isOverhead());
 	}
+
+	private void setPrayerOrbText(String text)
```

Comment:
This should also be reset on plugin shutdown.

Reply:
I'm not sure i understand, the prayer orb text has to be changed every game tick to show time so when the plugin is turned off, it resumes to use prayer points. Setting it explicitly on shutdown feel unnecessary (?)

Reply:
Does the client set that text each tick via some script? If so this is fine.

Reply:
Upon closer inspection it seems like script 82 sets it on prayer turn off or when you lose a prayer point, so I have used too draining prayers when testing.